docs/actuator-api-spec/backup-management-api.yaml

```
@@ -0,0 +1,185 @@
+      description: |
+        Returns a list of all available backups with their state and additional info,
+        sorted in descending order of backupId.
+      responses:
+        '200':
+          $ref: '#/components/responses/BackupList'
+        '500':
+          description: Internal Error
+          content:
+            text/plain:
+              schema:
+                type: string
+
+components:
+  responses:
+    BackupList:
+      description: |
+       List of all available backups.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/BackupList'
+
+  schemas:
+    BackupId:
+      title: Backup ID
+      description: |
+        The ID of the backup. The ID of the backup must be a positive numerical value. As backups
+        are logically ordered by their IDs (ascending), each successive backup must use a higher
+        ID than the previous one.
+      type: number
+      example: 1
+      minimum: 0
+
+    PartitionId:
+      title: Partition ID
+      description: |
+        The ID of a partition. This is always a positive number greater than or equal to 1.
+      type: number
+      minimum: 1
+      example: 3
+
+    StateCode:
+      title: Backup State
+      description: The state of the backup.
+      type: string
+      enum:
+        - DOES_NOT_EXIST
+        - IN_PROGRESS
+        - COMPLETED
+        - FAILED
+        - INCOMPLETE
+      example: IN_PROGRESS
+
+    BackupList:
+      title: List of backups
+      description: List of backups with their state and additional info
+      type: array
+      items:
+        $ref: '#/components/schemas/BackupInfo'
+
+    BackupInfo:
+      title: Backup Info
+      description: |
+        Detailed status of a backup. The aggregated state is computed from the backup state of each partition as:
+        - If the backup of all partitions is in state 'COMPLETED', then the overall state is 'COMPLETED'.
+        - If one is 'FAILED', then the overall state is 'FAILED'.
+        - Otherwise, if one is 'DOES_NOT_EXIST', then the overall state is 'INCOMPLETE'.
+        - Otherwise, if one is 'IN_PROGRESS', then the overall state is 'IN_PROGRESS'.
+      type: object
+      properties:
+        backupId:
+          readOnly: true
+          allOf:
+            - $ref: '#/components/schemas/BackupId'
+        state:
+          readOnly: true
+          allOf:
+            - $ref: '#/components/schemas/StateCode'
+        failureReason:
+          description: Reason for failure if the state is 'FAILED'
+          type: string
+          example: ""
+        details:
+          readOnly: true
+          description: |
+            Detailed list of the status of the backup per partition. It should always contain all
+            partitions known to the cluster.
+          type: array
+          items:
+            allOf:
+              - $ref: '#/components/schemas/PartitionBackupInfo'
+      required:
```

:wrench: I suspect that we will always return details
```suggestion
      required:
        - details
```